src/template/render.rs

```diff
@@ -471,9 +473,45 @@ impl<'a> ChainTemplateSource<'a> {
         };
         debug_span!("Executing command", ?command)
```

Comment:
This indentation is pretty ugly now, do you mind changing this?
```suggestion
        let _span = debug_span!("Executing command", ?command).entered();
```
The span covers the rest of the function, so we can just let it drop at the end of the function to exit.